W-13498888-(W-13498827)-proxy-settings-update (#181)
mulesoft/docs-rpa: rpa-builder/6.3/modules/ROOT/pages/install-and-configure.adoc

+
See xref:rpa-manager::usermanagement-manage.adoc#assign-privileges-to-a-user[Assign Privileges to a Users] for instructions. If you are in doubt, contact your organization's RPA administrator.

[[connect-with-anypoint]]
== Connect RPA Builder with RPA Manager (Anypoint Platform)

After opening RPA Builder for the first time, configure the connection with RPA Manager by using your Anypoint Platform credentials:

. Provide your RPA Manager instance address.
+

> image:anypoint-rpa-org-log-in.png[The Anypoint Platform login screen asking to configure the RPA Manager address, 35%, 35%]

Reviewer: ```suggestion
image:anypoint-rpa-org-log-in.png[The Anypoint Platform login screen prompt to configure the RPA Manager address, 35%, 35%]
```

+
*Proxy Settings* enable you to configure a specific proxy to use when stablishing the connection with Anypoint Platform.
+
By default, *Use System Proxy Settings* is enabled so that RPA Builder uses the same proxy configuration your system uses to access internet.
. Log in using your Anypoint Platform account. 
+

> image:anypoint-rpa-user-log-in.png[The Anypoint Platform login screen asking to configure the login credentials, 35%, 35%]

Reviewer: Line 21

After you log in, RPA Builder remains connected using the specified Anypoint Platform credentials. 

To connect to a different RPA Manager instance or use different log in credentials:

. In the top menu bar, navigate to *Options > Account*.
. Click *Sign out*.

> . Provide the new RPA Manager address and log in credentials. 

Reviewer: ```suggestion
. Provide the new RPA Manager address and login credentials. 
```

[[connect-with-rpa-manager]]
== Connect RPA Builder with RPA Manager (Legacy)

> If your organization was not migrated to Anypoint Platform yet, provide your RPA Manager user credentials to stablish the connection after opening RPA Builder for the first time:

Reviewer: ```suggestion
If your organization is not migrated to Anypoint Platform, provide your RPA Manager user credentials to establish the connection after opening RPA Builder for the first time:
```

> . In the top menu bar, select *Options* > *Repository*. (This menu opens automatically the first time you open RPA Builder)

Reviewer: Do you need the parenthetical or is it obvious from the UI?

The RPA Manager password that RPA Builder uses to log in.
** *Proxy Settings*
+

> Enables you to configure a specific proxy to use when connecting with RPA Manager.

Reviewer: (Not sure if that changes the meaning)
